components/Side/ToggleMode.vue

```diff
@@ -0,0 +1,56 @@
+<script setup>
+import { ref } from "vue";
+
+const colorMode = useColorMode();
+const toggleChecked = ref(false);
+
+// Function to toggle the select dropdown
+const toggleSelect = () => {
+  colorMode.preference = toggleChecked.value ? "dark" : "light";
+};
+</script>
+<style lang="postcss">
+body {
+  @apply min-h-screen;
```

Comment:
Remove 
`<style lang="postcss">
body {
  @apply min-h-screen;
}
</style>`

Reply:
Without it I can't import ref and i got the following 
![image](https://github.com/kamilziemann/nuxt_crm/assets/92382444/8b0cf1f4-a2e3-40ba-889f-932b121a7461)

Reply:
As far as I know ref is globally imported in nuxt, so there is no need to import it in every component - should works without ref import

Reply:
After remove
```
<style lang="postcss">
body {
  @apply min-h-screen;
}
</style>
```
and `import { ref, onMounted } from "vue";` you have to restart vite server to avoid this error